### DISCUSSION TITLE
/vm/src/circuit/r1cs.rs: fn store_reg : build constraints

### DISCUSSION THREAD

**Discussion by hjlpb** [nexus-xyz/nexus-zkvm#199] [General]
<img width="655" alt="image" src="https://github.com/nexus-xyz/nexus-zkvm/assets/20498177/7e3b869e-9b2a-4dfb-91c9-ea89b84e7262">

**Comment by ghost** [comment]
I think there's something to fix here. The original version doesn't mention `x1` `x2` through `x32` variables without `'`s. And the proposed fix makes sense to me.

**Comment by ghost** [comment]
@hjlpb Do you want to file a pull-request for this change?

**Comment by sjudson** [comment]
Closing w/ merged PR to fix: https://github.com/nexus-xyz/nexus-zkvm/pull/205

**Comment by taint160596** [comment]
Thank you
